WMFData/Sources/WMFData/Data Controllers/Year In Review/WMFYearInReviewDataController.swift

```diff
@@ -292,4 +293,84 @@ public class WMFYearInReviewDataController {
         
         return true
     }
+    
+    public func fetchUserContributionsCount(username: String, languageCode: String, completion: @escaping (Result<(Int, Bool), Error>) -> Void) {
```

Comment:
@l-olson1214 For this Part 1 PR, can you only include this method and remove the other changes? I'm going for a different approach where the actual data will be fetched upon app resume (see [here](https://github.com/wikimedia/wikipedia-ios/pull/5024/files#diff-6eda737a0f8aa712d064e24a5159ac83bd352538048c5fbe777ded61e08d082dR617) in my saved article personalized slide [PR](https://github.com/wikimedia/wikipedia-ios/pull/5024)) and saved to a persisted report object. We will then reference that object when we create the `YearInReviewSlideContent`, so there won't be any need for marking the slide view models as `@Published` and updating them in the coordinator.

Once my PR is merged, you'll be unblocked to do parts 2 and 3 of this task.
You will be able to call this method from [here](https://github.com/wikimedia/wikipedia-ios/pull/5024/files#diff-1c01e90351b11ab077c5dff447720470dad032889b5206d5e9c5e4a69065649cR186), update and save the report object, then reference and create the slide [here](https://github.com/wikimedia/wikipedia-ios/pull/5024/files#diff-72caf34090dc13194bbfc66eeb449b72948cef8bd240c2a17c71437894632f90R109).

Reply:
🫡 Done and done :)